databases/week1/data.sql

```diff
@@ -0,0 +1,47 @@
+-- 1 task
+SELECT count(id)
+FROM task
+-- 2 task
+SELECT title, due_date
+FROM task
+WHERE due_date IS NULL;
+-- 3 task
+SELECT *
+FROM TASK
+WHERE STATUS_ID = 3
+-- 4 task
+SELECT *
+FROM TASK
+WHERE STATUS_ID != 3
+-- 5 task
+SELECT *
+FROM TASK
+ORDER BY created ASC
+-- 6 task
+SELECT *
+FROM TASK
+ORDER BY created DESC
+-- LIMIT 1
+--Alternate for limit
+-- SELECT TOP 1 *
+-- FROM TASK
+-- ORDER BY created DESC
+-- 7 task
+SELECT title, due_date
+FROM TASK
+WHERE title LIKE '%database%' OR description LIKE '%database%';
+-- 8 task
+SELECT TASK.TITLE, STATUS.NAME
+FROM TASK, STATUS
```

Comment:
Try to do by JOIN statement

Reply:
Done ! Need some more understanding on Joins.  :)